index.js

```diff
@@ -171,7 +171,7 @@ const getFirefoxWithFallbackOnOSX = function () {
 
   const firefoxDirNames = Array.prototype.slice.call(arguments)
   const prefix = '/Applications/'
-  const suffix = '.app/Contents/MacOS/firefox-bin'
+  const suffix = '.app/Contents/MacOS/firefox'
```

Comment:
For even a better stability over time and to have code that is not tight to a specific browser executalbe it would be probably better to read out the `Info.plist` file for the executable bundle. We now do that in geckodriver as well.

Reply:
Ack, perhaps in a later change. Doing that synchronously, and without new dependencies, and such that we can still cleanly decide between multiple optional applications, will be non-trivial. Also considering that Karma is deprecated and in maintenance mode basically, it will be hard to get a maintainer to review and land this swiftly as-is.